tests/lib/openstack_query/handlers/test_client_side_handler.py

```diff
@@ -0,0 +1,207 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from parameterized import parameterized
+
+from openstack_query.handlers.client_side_handler import ClientSideHandler
+from exceptions.query_preset_mapping_error import QueryPresetMappingError
+
+from tests.lib.openstack_query.mocks.mocked_query_presets import MockQueryPresets
+from tests.lib.openstack_query.mocks.mocked_props import MockProperties
+
+
+class ClientSideHandlerBaseTests(unittest.TestCase):
```

Comment:
Aside: what's your thoughts on switching this project to `pytest in the future`? Since we get fixtures / paramterize etc.

Reply:
I wanted to use `pytest` - but all the other tests for stackstorm defaulted to using `nose.tools` and `unittest` - so I stuck with that. 

When we move this out - we may want to refactor to use `pytest`